.bazelrc

```diff
@@ -124,13 +124,12 @@ build:remote --auth_enabled=true
 build:remote --experimental_inmemory_jdeps_files
 build:remote --experimental_inmemory_dotd_files
 build:remote --experimental_remote_download_outputs=toplevel
-test:remote --experimental_remote_download_outputs=minimal
```

Comment:
QQ: Will this increase costs for intermediate steps? Is there a way to just get the logs which is presumably what you need for the output? (I guess why did this change?)

Reply:
No, this was my ask in https://github.com/bazelbuild/bazel/issues/8934 to make `bazel test` with toplevel only to download logs and summary (test.xml). So now we get back the AZP test summary: https://dev.azure.com/cncf/envoy/_build/results?buildId=10515&view=ms.vss-test-web.build-test-results-tab

Reply:
Ah nice. Awesome!